ens/src/lib.rs

```diff
@@ -287,7 +287,7 @@ async fn setup(
         }
     }
 
-    let call = resolver.multicall(calls)?.gas(21000);
+    let call = resolver.multicall(calls)?;
```

Comment:
You tested this? It wasn't working without previously no?

Reply:
Yes, it's tested, this should work fine now.